### PR TITLE
[FLINK-20998][docs] - flink-raw.jar no longer mentioned + prettier maven xml dependencies

### PR DESCRIPTION
**Pull request by sv3ndk**
## What is the purpose of the change

*fix documentation mistake: the raw format is a built-in format distributed with blink-table-planner but was documented as a separate jar.*

## Brief change log

  - *removed `flink-raw.jar` reference from `site-connector.yml`*
  - *update the raw format documentation: no longer loads the `sql-connnector-download-table.html` snippet*
  - *added maven xml format to connectors and format documentation for user convenience*

## Verifying this change

I manually ran `./build_docs.sh -p  `, the format seems ok now.

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Review by wuchong — COMMENTED** [review]
I think adding maven xml format is a good idea. But the table structure is not very clear because the maven cell is expanded now. I'm thinking about adding table border to it, like following, what do you think?

**Before:**
image

**After:**
image

Besides, we should also update the Chinese version of the `sql-connnector-download-table.html`. 

I upload a commit to my repo, you can have a look. https://github.com/wuchong/flink/commit/b3159626af77adba08143278d151c76e28b7dc01

File: docs/dev/table/connectors/formats/raw.zh.md [hunk 1]
```diff
@@ -33,13 +33,7 @@ Raw format 允许读写原始（基于字节）值作为单个列。
...
 注意: 这种格式将 `null` 值编码成 `byte[]` 类型的 `null`。这样在 `upsert-kafka` 中使用时可能会有限制，因为 `upsert-kafka` 将 `null` 值视为 墓碑消息（在键上删除）。因此，如果该字段可能具有 `null` 值，我们建议避免使用 `upsert-kafka` 连接器和 `raw` format 作为 `value.format`。
 
-依赖
-------------
-
-{% assign connector = site.data.sql-connectors['raw'] %}
-{% include sql-connector-download-table.zh.html
-    connector=connector
-%}
+Raw format 连接器是内置的。
```
[the remark this answers is not in the document]
Comment (sv3ndk) [reply docs/dev/table/connectors/formats/raw.zh.md:36]: > I'm thinking about adding table border to it, like following, what do you think?
That's much clearer indeed, thanks for the suggestion. 

> we should also update the Chinese version of the sql-connnector-download-table.html.
Sorry for having overlooked that one, and thanks for the commit, I copied here your update.

> Cool translation ;)
Thanks. I copy-pasted it from the "DataGen" connector, I don't speak Chinese myself unfortunately.

**Comment by sv3ndk** [comment]
Hi,
Do you need more input from me on this one?

**Comment by wuchong** [comment]
@sv3ndk , sorry, I do think I missed to merge this PR. 

Could you help to open a pull request for release-1.12 ?

**Review by wuchong — APPROVED** [review]
LGTM.

**Comment by sv3ndk** [comment]
Thanks for the merge @wuchong 

> Could you help to open a pull request for release-1.12 ?

Gladly. I'm new here so I want to make sure I do this correctly: all I should do is open another PR with the same commits to the branch called release-1.12, is this correct?

**Comment by wuchong** [comment]
Yes, you are right. You can just cherry pick the merged commit to your dev branch based on latest release-1.12 branch. But you may need to resolve conflicts if there are conflicts.

**Comment by sv3ndk** [comment]
There you go: https://github.com/apache/flink/pull/14723

**Pull request merged**